### PR TITLE
feat: injective chain support

### PR DESCRIPTION
**Pull request by debendraoli**
This pull request includes refactoring of the code and fixing some typos in the comments and command flags. The changes improve code consistency and readability.

It also adds support for injective chain

### CODE REVIEW

File: relayer/chains/wasm/provider.go [hunk 1]
```diff
@@ -225,11 +225,12 @@ func (p *Provider) prepareAndPushTxToMemPool(ctx context.Context, acc, seq uint6
 		WithSequence(seq)
 
 	if txf.SimulateAndExecute() {
-		_, adjusted, err := p.client.EstimateGas(txf, msgs...)
+		tx, adjusted, err := p.client.EstimateGas(txf, msgs...)
 		if err != nil {
 			return nil, err
 		}
 		txf = txf.WithGas(adjusted)
+		fmt.Println(tx.GasInfo)
```
Comment (bcsainju) [inline relayer/chains/wasm/provider.go:233]: remove the println statement.

File: relayer/chains/wasm/client.go [hunk 2]
```diff
@@ -125,17 +126,23 @@ func (c *Client) GetAccountInfo(ctx context.Context, addr string) (sdkTypes.Acco
 
 	var account sdkTypes.AccountI
 
+	fmt.Println(c.ctx.Keyring.List())
```
Comment (bcsainju) [inline relayer/chains/wasm/client.go:129]: remove the print with logging statement if required.

File: relayer/chains/wasm/types/const.go [hunk 3]
```diff
@@ -3,15 +3,17 @@ package types
...
+	"github.com/cosmos/relayer/v2/relayer/codecs/injective"
 )
 
 const (
-	CodeTypeOK  uint32 = 0
-	CodeTypeErr uint32 = 1
-	ChainType          = "cosmos"
+	CodeTypeOK       uint32 = 0
+	CodeTypeErr      uint32 = 1
+	ethereumCoinType        = uint32(60)
```
Comment (AntonAndell) [inline relayer/chains/wasm/types/const.go:12]: Is this used anywhere?

File: relayer/chains/wasm/client.go [hunk 4]
```diff
@@ -128,14 +128,16 @@ func (c *Client) GetAccountInfo(ctx context.Context, addr string) (sdkTypes.Acco
...
 	return account, nil
 }
 
 // Create new AccountI
 func (c *Client) CreateAccount(uid, passphrase string) (string, string, error) {
 	// create hdpath
-	kb := keyring.NewInMemory(c.ctx.Codec)
+	kb := keyring.NewInMemory(c.ctx.Codec, func(options *keyring.Options) {
+		options.SupportedAlgos = types.SupportedAlgorithms
+		options.SupportedAlgosLedger = types.SupportedAlgorithmsLedger
```
[the remark this answers is not in the document]
Comment (debendraoli) [reply relayer/chains/wasm/client.go:139]: nope. added for injective edge cases.

**Review by AntonAndell — DISMISSED** [review]
I think everything looks good. As long as it has been tested with injective i see no real faults from the code.

File: relayer/chains/evm/route.go [hunk 5]
```diff
@@ -63,7 +63,7 @@ func (p *Provider) SendTransaction(ctx context.Context, opts *bind.TransactOpts,
 
 	opts.GasLimit = gasLimit + (gasLimit * p.cfg.GasAdjustment / 100)
 
-	p.log.Info("gas info", zap.Uint64("gas_limit", opts.GasLimit), zap.Uint64("gas_price", opts.GasPrice.Uint64()))
+	p.log.Info("gas info", zap.Uint64("gas_limit", gasLimit), zap.Uint64("gas_price", opts.GasPrice.Uint64()))
```
Comment (AntonAndell) [inline relayer/chains/evm/route.go:66]: Why do we debug the estimate gas rather than the limit we will use?

File: relayer/chains/evm/provider.go [hunk 6]
```diff
@@ -115,7 +115,7 @@ func (p *Config) Validate() error {
 
 func (p *Config) sanitize() error {
 	if p.GasAdjustment == 0 {
-		p.GasAdjustment = 50
+		p.GasAdjustment = 5
```
Comment (AntonAndell) [inline relayer/chains/evm/provider.go:118]: Why use 5, is it meant to be 5% default?

**Review by AntonAndell — APPROVED** [review]
LGTM

**Pull request merged**